objectivec/GPBWellKnownTypes.h

```diff
@@ -96,4 +115,80 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+#pragma mark - GPBAny
+
+/**
+ * Category for GPBAny help work with the message within the object.
+ **/
+@interface GPBAny (GBPWellKnownTypes)
+
+/**
+ * Serializes message and returns a new instance to contain the info. Uses the
```

Comment:
Returns a new instance of GPBAny that includes the serialized message.

below too